<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(47, 47, 47, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/ASUS/OneDrive/เดสก์ท็อป/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btresever">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>490</y>
      <width>235</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(188, 125, 0, 220);
font: 57 18pt &quot;Mali Medium&quot;;
border-radius:10;</string>
    </property>
    <property name="text">
     <string>RESEVER [จองโต๊ะ]</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablewidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>631</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 220);</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>tel</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>zone</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="listtext">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>-20</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>28</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 57 28pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;List :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btaboutus">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>590</y>
      <width>235</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>18</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(188, 125, 0, 220);
font: 57 18pt &quot;Mali Medium&quot;;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>ABOUT us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btdelete_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(167, 167, 167);

font: 57 18pt &quot;Mali Medium&quot;;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="idtext_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 57 18pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>ID : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btdelete_4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>370</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);
font: 57 18pt &quot;Mali Medium&quot;;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idinput_2">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="listtext_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>28</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 57 28pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Edit List&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idinput_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>170</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="namefix">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 57 18pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>NAME : </string>
    </property>
   </widget>
   <widget class="QLabel" name="namefix_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>220</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 57 18pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>TEL : </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idinput_4">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>230</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idinput_5">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>290</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="namefix_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>280</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 57 18pt &quot;Mali Medium&quot;;</string>
    </property>
    <property name="text">
     <string>ZONE : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btdelete_5">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mali Medium</family>
      <pointsize>14</pointsize>
      <weight>7</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(167, 167, 167);
font: 57 14pt &quot;Mali Medium&quot;;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>checkID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btdelete_6">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>370</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(85, 85, 127);
font: 57 18pt &quot;Mali Medium&quot;;
border-radius:10;
</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <zorder>namefix_3</zorder>
   <zorder>namefix</zorder>
   <zorder>listtext</zorder>
   <zorder>logo</zorder>
   <zorder>btresever</zorder>
   <zorder>tablewidget</zorder>
   <zorder>btaboutus</zorder>
   <zorder>btdelete_3</zorder>
   <zorder>idtext_2</zorder>
   <zorder>btdelete_4</zorder>
   <zorder>idinput_2</zorder>
   <zorder>listtext_2</zorder>
   <zorder>idinput_3</zorder>
   <zorder>namefix_2</zorder>
   <zorder>idinput_4</zorder>
   <zorder>idinput_5</zorder>
   <zorder>btdelete_5</zorder>
   <zorder>btdelete_6</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
